asyncpg/types.py

```diff
@@ -85,6 +85,52 @@ def upper_inf(self):
     def isempty(self):
         return self._empty
 
+    def issubset(self, other):
+        if self._empty:
+            return True
+        if other._empty:
+            return False
+
+        if other._lower is None:
+            lowerOk = True
+        else:
+            if self._lower is None:
+                return False
+            if other._lower_inc or not self._lower_inc:
+                lowerOk = self._lower >= other._lower
+            else:
+                lowerOk = self._lower > other._lower
+
+        if not lowerOk:
+            return False
+
+        if other._upper is None:
+            upperOk = True
+        else:
+            if self._upper is None:
+                return False
+            if other._upper_inc or not self._upper_inc:
+                upperOk = self._upper <= other._upper
+            else:
+                upperOk = self._upper < other._upper
+
+        return lowerOk and upperOk
```

Comment:
`lowerOk` is always `True` by this point, so returning just `upperOk` is sufficient.